packages/contracts/contracts/newsroom/Newsroom.sol

```diff
@@ -2,18 +2,14 @@ pragma solidity ^0.4.19;
 import "./ACL.sol";
 
 contract Newsroom is ACL {
-  event ContentProposed(address indexed author, uint indexed id);
-  event ContentApproved(uint id);
-  event ContentDenied(uint id);
+  event ContentAdded(address indexed editor, uint indexed id);
```

Comment:
Added -> Published might be more clear? Also might be worth it to add a bit more info to this event (url would be helpful)

Reply:
RevisionPublished? Then we can have a separate event that fires in addition to this one if the article is brand new, as opposed to just a revision